src/js/service/UnitCountMachine.js

```diff
@@ -0,0 +1,94 @@
+import { ALERT_MESSAGE, VENDING_MACHINE_CONSTANT } from './constant.js';
+import ValidationError from './ValidationError.js';
+import { isInsertedCoinsValid } from './validator.js';
+
+/**
+ * @typedef {Object} UnitCountInfo
+ * @property {number} amount
+ * @property {Object.<string, number>} unitInfo
+ */
+
+export default class UnitCountMachine {
+  /**@type {UnitCountInfo} */
+  #unitCountInfo;
+
+  constructor() {
+    this.#unitCountInfo = {
+      amount: 0,
+      unitInfo: VENDING_MACHINE_CONSTANT.CHANGES.UNITS.reduce(
+        (result, unit) => ({
+          ...result,
+          [unit]: 0,
+        }),
+        {}
+      ),
+    };
+  }
+
+  /**
+   *
+   * @returns {UnitCountInfo}
+   */
+  get unitCountInfo() {
+    return this.#unitCountInfo;
+  }
+
+  /**
+   *
+   * @returns {number[]}
+   */
+  getUnits() {
+    return Object.keys(this.#unitCountInfo.unitInfo).sort((a, b) => Number(b) - Number(a));
+  }
```

Comment:
```suggestion
  /**
   *
   * @returns {number[]}
   */
  get units() {
    return Object.keys(this.#unitCountInfo.unitInfo).sort((a, b) => Number(b) - Number(a));
  }
```
이럴 때 get이 쓰여야 하는게 아닐까요!?

Reply:
제안해주신 코드는 수동으로 반영할게요!
PR 올리고 가끔 충돌이 나는데 수정하는 작업이 좀 번거롭습니다!